src/main.rs

```diff
@@ -38,7 +40,11 @@ enum TestEvent {
     #[serde(rename = "ok")]
     Ok { name: String },
     #[serde(rename = "failed")]
-    Failed { name: String, stdout: String },
+    Failed {
+        name: String,
+        stdout: Option<String>,
+        stderr: Option<String>,
```

Comment:
I don't think Rust includes `stderr` separately from `stdout`.

Reply:
I agree @jstarks, libtest in `cargo test` does not send a separate stderr, but we use a different test framework which provides both `stdout` and `stderr` and we tried to use cargo2junit for it as well.
In the end this proved to be much more work than desired, due to transitive dependencies to `junit-report`, and in the end we gave up and just re-implemented direct junit xml dumping from scratch.
If you think this hurts the project, please feel free to remove the lines.